src/main/java/htsjdk/samtools/cram/index/CRAIEntry.java

```diff
@@ -11,11 +11,7 @@
  * A class representing CRAI index entry: file and alignment offsets for each slice.
  * Created by vadim on 10/08/2015.
  */
-public class CRAIEntry implements Comparable<CRAIEntry> {
-    private final int sequenceId;
```

Comment:
these 3 fields are now the new class `CRAIQuery` because that is all that is needed for querying.